lib/documentLoader.js

```diff
@@ -3,147 +3,55 @@
  */
 'use strict';
 
-const {agent} = require('bedrock-https-agent');
+const didContext = require('did-context');
+const ed25519 = require('ed25519-signature-2020-context');
+const x25519 = require('x25519-key-agreement-2020-context');
+const cred = require('credentials-context');
+const veresOneCtx = require('veres-one-context');
+
 const bedrock = require('bedrock');
+
+// Includes support for did:key and did:v1
 const {didIo} = require('bedrock-did-io');
 const {
-  documentLoader: bedrockLoader
+  jsonLdDocumentLoader,
+  httpClientHandler
 } = require('bedrock-jsonld-document-loader');
-const {httpClient} = require('@digitalbazaar/http-client');
-const jsonld = require('jsonld');
-const {constants: {DID_CONTEXT_URL}} = require('did-context');
-const {constants: {VERES_ONE_CONTEXT_V1_URL}} = require('veres-one-context');
 
 const {config} = bedrock;
+const {contexts: credentialsContext, constants: {CREDENTIALS_CONTEXT_V1_URL}} =
+  cred;
+
+jsonLdDocumentLoader.addStatic(ed25519.constants.CONTEXT_URL,
```

Comment:
@dlongley @mattcollier 
I think it might make sense for us to add these common contexts to `bedrock-jsonld-document-loader`. (So that we don't have to keep re-initializing them in our various libs.)

Reply:
n/m, I see we're loading them via bedrock- modules.